Makefile

```diff
@@ -111,70 +117,56 @@ PROTOC = $(BIN)/protoc
 BINS = $(DOCKER_COMPOSE) $(GLIDE) $(THRIFT) $(PROTOC) $(BIN)/thriftrw-plugin-yarpc $(BIN)/protoc-gen-yarpc-go
 
 $(DOCKER_COMPOSE):
-	mkdir -p $(BIN)
+	@mkdir -p $(BIN)
```

Comment:
Any reason to hide all this output?

Trying to manage what info is useful to look at?